build.gradle

```diff
@@ -125,7 +125,7 @@ dependencies {
   compile 'commons-codec:commons-codec:1.10'
 
   // Test dependencies
-  testCompile('com.xebialabs.cloud:overcast:1.2.1') {
+  testCompile('com.xebialabs.cloud:overcast:2.5.1') {
     exclude group: 'com.xebialabs.overthere', module: 'overthere'
```

Comment:
This is to prevent a circular dependency. Does the new overcast still include overthere as a compile dependency?